app/routes.py

```diff
@@ -86,7 +83,7 @@ def create_shortlink():
     """
     logger.debug("Shortlink Creation route entered at %f", time.time())
     if request.headers.get('Origin') is None or not \
-            re.match(allowed_domains_pattern, request.headers['Origin']):
+            re.match(ALLOWED_DOMAINS_PATTERN, request.headers['Origin']):
         logger.critical(
             "Shortlink Error: Invalid Origin. ( %s )",
             request.headers.get('Origin', 'No origin given')
```

Comment:
This check is not needed as the origin is now check in the `before_request` hook.